tests/cypress/integration/issue_1919_check_text_attr.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check label attribute changes', () => {
+
+    const issueId='1919'
+    const labelName=`Issue ${issueId}`
+    const taskName=`New annotation task for ${labelName}`
+    const attrName=`Attr for ${labelName}`
+    const textDefaultValue='Some default value for type Text'
+    const image=`image_${issueId}.png`
+    const newLabelAttrValue = 'New attribute value'
+    const width='800'
+    const height='800'
+    const posX=10
+    const posY=10
+    const color='gray'
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.get('[type="submit"]').click()
+        cy.imageGenerator('cypress/fixtures', image, width, height, color, posX, posY, labelName)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, image)
+    })
+
+    describe(`Open task "${taskName}"`, () => {
+        it('The annotation task present in the list', () => {
+            cy.contains('strong', taskName)
+            .parent()
+            .parent()
+            .parent()
+            .contains('a', 'Open').click()
```

Comment:
Can we get this button in a simpler way? ``cy.get('[href="/tasks/${taskID}"]').click()``

Reply:
`${taskID}` unknown in advance. I tried using a slightly different solution.